packages/p2p/src/stores/sendbird-store.ts

```diff
@@ -371,16 +371,12 @@ export default class SendbirdStore extends BaseStore {
             { fireImmediately: true }
         );
 
-        this.disposeActiveChatChannelReaction = reaction(
-            () => this.active_chat_channel,
-            (active_chat_channel?: GroupChannel) => {
-                if (active_chat_channel) {
```

Comment:
if its not the active_chat_channel shouldn't we still clear the channel messages? what would happen if we switch between an ongoing chat and then you go to a new chat, will it show the previous messages?

Reply:
Let me test this @ameerul-deriv

Reply:
@jim-deriv  wasn't this issue happening because it was not getting fired at the initial step? can we try adding `fireimmediately: true`.. i think that would fix the issue

Reply:
@nada-deriv, as I checked, It doesn't fire, because the value of `this.active_chat_channel` doesn't change after we leave the tab and come back (It remains set to the first value that was assigned to it when we navigated to the page in the first place). I'll try out your suggestion regardless. Thank you